components/common/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+import { Container } from 'reactstrap';
+
+export const AnnouncementBannerSection = styled.div`
```

Comment:
Nitpick: Juha and I agreed to started prefixing styled component names with `Styled`, i.e. `StyledAnnouncementBannerSection`. It makes names a bit verbose but is really handy to easily spot "real" components and styled components in the render function

Reply:
https://github.com/kausaltech/kausal-watch-ui/pull/243/commits/343691cd199f6f6e02ac8f28526d0760fb084fc5